raster-foundry/raster-foundry
# Add shape browse filter

## Overview
Add shape browse filter
Support filtering rf and planet repos by shape
- rf is disabled until the api supports it
- planet is fully functional

### Checklist

~- [ ] Styleguide updated, if necessary~
~- [ ] Swagger specification updated, if necessary~
~- [ ] Symlinks from new migrations present or corrected for any new migrations~

## Testing Instructions

* Verify that the planet repo AOI filter works
  * note that thumbnails are not fixed on this pr, so some scenes that look like they don't intersect the shapes do in fact touch
* Test draw and edit functionality, try to break it
* Uncomment the shape filter in the raster foundry repo service and verify that it sends the shape to the api as a uuid
* Verify that adding a shape filter disables the bbox filter and moving the map view does not cause the scene list to reload
* Verify that adding a shape filter causes it to show up on the map

Closes #2951 
Closes #2892 
Depends on https://github.com/raster-foundry/raster-foundry/pull/2937 (rebase on develop when merged)
